src/plugins/intel_cpu/src/nodes/common/cpu_convert.cpp

```diff
@@ -784,6 +1065,9 @@ bool is_supported_convert(ov::element::Type srcPrc, ov::element::Type dstPrc) {
     OV_SWITCH(intel_cpu, isSupported, ctx, std::tie(srcPrc, dstPrc), INTEL_CPU_CVT_FROM_BIN_LIST);
     OV_SWITCH(intel_cpu, isSupported, ctx, std::tie(srcPrc, dstPrc), INTEL_CPU_CVT_FROM_4BIT_LIST);
     OV_SWITCH(intel_cpu, isSupported, ctx, std::tie(srcPrc, dstPrc), INTEL_CPU_CVT_FROM_BYTE_FP_LIST);
+#if defined(OPENVINO_ARCH_X86_64)
+    OV_SWITCH(intel_cpu, isSupported, ctx, std::tie(srcPrc, dstPrc), INTEL_CPU_CVT_FP8_LIST);
```

Comment:
INTEL_CPU_CVT_FP8_LIST always in INTEL_CPU_CVT_LIST on line 1067. Then always support fp8 conversion regardless OPENVINO_ARCH_X86_64 or not.

Reply:
Good point! Removed. Thanks Chenhu!